recipes-bsp/trusted-firmware-a/trusted-firmware-a.bb

```diff
@@ -0,0 +1,77 @@
+DESCRIPTION = "Trusted Firmware-A for Renesas RZ"
+
+require include/rzg2l-optee-config.inc
+inherit deploy
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302 \
+"
+# Set S variable to folder that includes Makefile
+S = "${WORKDIR}/git"
+
+SRC_URI:rzg2l-sbc = " \
+    git://github.com/Renesas-SST/rz-atf.git;name=machine;branch=${BRANCH};protocol=https \
+"
+
+BRANCH:rzg2l-sbc = "scarthgap/rz-sbc"
```

Comment:
It should be styhead branch instead of scarthgap.

Reply:
In the git repository https://github.com/Renesas-SST/rz-atf , I can only find scarthgap branch.

Reply:
You can create a styhead branch for the styhead layer.

Reply:
I have created styhead branch on repository https://github.com/Renesas-SST/rz-atf . 
Updated in the commit : dcf019b3e552b7cfa96e5d5f6d37e831a607ed34